includes/User.php

```diff
@@ -3188,19 +3167,19 @@ public function saveSettings() {
 		$dbw = wfGetDB( DB_MASTER );
 		$dbw->update( 'user',
 			array( /* SET */
-				'user_name' => $this->mName,
-				'user_password' => $this->mPassword,
-				'user_newpassword' => $this->mNewpassword,
-				'user_newpass_time' => $dbw->timestampOrNull( $this->mNewpassTime ),
-				'user_real_name' => $this->mRealName,
-				'user_email' => $this->mEmail,
-				'user_email_authenticated' => $dbw->timestampOrNull( $this->mEmailAuthenticated ),
-				'user_touched' => $dbw->timestamp( $this->mTouched ),
-				'user_token' => strval( $this->mToken ),
-				'user_email_token' => $this->mEmailToken,
-				'user_email_token_expires' => $dbw->timestampOrNull( $this->mEmailTokenExpires ),
+				   'user_name' => $this->mName,
+				   'user_password' => $this->mPassword,
+				   'user_newpassword' => $this->mNewpassword,
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Define a constant instead of duplicating this literal "user_newpassword" 3 times. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1192)